dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/ConsoleMonitorThread.java

```diff
@@ -197,6 +205,13 @@ public void streamAppended(String text, IStreamMonitor monitor) {
                 while (st.hasMoreTokens()) {
                     String s = st.nextToken();
 
+                    if (s.startsWith("{")) {
```

Comment:
Is the token (eg. String s) the entire JSON?   Or just one line?  eg.  wondering if the one token can be just { because it is on one line.

Reply:
Each JSON object is on a single line.  I already checked with Don Bourne that this is a safe thing to do.

{"ibm_datetime":"2017-11-24T16:59:28.279-0500","host":"9.85.139.109","ibm_userDir":"C:\/home\/work\/lwas_open_jsonlog\/wlp\/usr\/","ibm_serverName":"defaultServer","ibm_sequence":"1511560768279_0000000000001","loglevel":"AUDIT","ibm_messageId":"CWWKE0001I","module":"com.ibm.ws.kernel.launch.internal.FrameworkManager","ibm_threadId":"00000001","message":"CWWKE0001I: The server defaultServer has been launched."}
{"ibm_datetime":"2017-11-24T16:59:29.420-0500","host":"9.85.139.109","ibm_userDir":"C:\/home\/work\/lwas_open_jsonlog\/wlp\/usr\/","ibm_serverName":"defaultServer","ibm_sequence":"1511560769420_0000000000002","loglevel":"INFO","ibm_messageId":"CWWKE0002I","module":"com.ibm.ws.kernel.launch.internal.FrameworkManager","ibm_threadId":"00000001","message":"CWWKE0002I: The kernel started after 1.29 seconds"}

Reply:
I suppose it depends on where 'message' is? 
.....{ "message": "value" }......
or
{
   "message": "value",
}